lib/bundles/inspec-compliance/cli.rb

```diff
@@ -18,78 +18,67 @@ def self.subcommand_prefix
-        # TODO: we should login with the refreshtoken here
-      elsif !options['refresh_token'].nil?
-        _success, msg = login_refreshtoken(url, options)
-      else
-        puts 'Please run `inspec compliance login SERVER` with options --token or --refresh_token, --user, and --insecure or --not-insecure'
-        exit 1
-      end
-
-      puts '', msg
+      Compliance::API.login(options)
     end
 
-    desc "login_automate SERVER --insecure --user='USER' --ent='ENT' --usertoken='TOKEN'", 'Log in to an Automate SERVER'
+    desc "login_automate https://SERVER --insecure --user='USER' --ent='ENTERPRISE' --usertoken='TOKEN'", 'Log in to a Chef Automate SERVER (DEPRECATED: Please use `login`)'
     long_desc <<-LONGDESC
-      `login_automate` allows you to use InSpec with Chef Automate
+      This commmand is deprecated and will be removed, please use `--login`.
+
+      `login_automate` allows you to use InSpec with Chef Automate.
 
-      You need to a user-token for communication with Chef Automate. More
-      information about token retrieval for Automate is available at:
-      https://docs.chef.io/api_delivery.html
+      You need to a token for communication. More information about token retrieval
+      is available at:
+        https://docs.chef.io/api_automate.html#authentication-methods
+        https://docs.chef.io/api_compliance.html#obtaining-an-api-token
     LONGDESC
-    option :dctoken, type: :string,
-      desc: 'Data Collector token'
-    option :usertoken, type: :string,
-      desc: 'Automate user token'
-    option :user, type: :string,
-      desc: 'Automate username'
-    option :ent, type: :string,
-      desc: 'Enterprise for Chef Automate reporting'
     option :insecure, aliases: :k, type: :boolean,
       desc: 'Explicitly allows InSpec to perform "insecure" SSL connections and transfers'
-    def login_automate(server) # rubocop:disable Metrics/AbcSize
+    option :user, type: :string, required: true,
+      desc: 'Username'
+    option :usertoken, type: :string, required: false,
+      desc: 'Access token (DEPRECATED: Please use `--token`)'
+    option :token, type: :string, required: false,
+      desc: 'Access token'
+    option :dctoken, type: :string, required: false,
+      desc: 'Data Collector token'
+    option :ent, type: :string, required: true,
+      desc: 'Enterprise for Chef Automate reporting'
+    def login_automate(server)
+      warn '[DEPRECATION] `inspec compliance login_automate` is deprecated. Please use `inspec compliance login`'
       options['server'] = server
-      url = options['server'] + '/compliance'
 
-      if url && !options['user'].nil? && !options['ent'].nil? && (!options['dctoken'].nil? || !options['usertoken'].nil?)
-        msg = login_automate_config(url, options['user'], options['dctoken'], options['usertoken'], options['ent'], options['insecure'])
-        puts '', msg
-        exit 0
+      if options['usertoken']
```

Comment:
Despite what was in the issue that preceded this PR, we probably don't need this deprecation here since the whole command is deprecated. As long as the `inspec compliance login` command doesn't accept a `--usertoken` flag, we're covered.